### PR TITLE
`General`: Scroll up when clicking on a course in course management

### PR DESCRIPTION
**Pull request by edkaya**
#### General

- [x] This is a small issue that I tested locally.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

It fixes the scroll progress in course managment when you click on one of the courses. The problem was when you select a course at the bottom most, you will be navigated to that course but the page does not start at top, but keeps the scroll progress. Detailed description and screen cast can be found at #8738. 

### Description

I added scrollTop function in the corrosponding component, so after each view initialization, it automatically scrolls to top. 

### Steps for Testing

Prerequisites:
- 1 Admin
- 3 Courses

1. Log in to Artemis with Admin account
2. Navigate to Course Management
3. Scroll down and select the bottom most course 
4. Verify that view starts at the top instead of where you left in the course management page.

#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### CODE REVIEW

File: src/main/webapp/app/course/manage/course-management-tab-bar/course-management-tab-bar.component.ts [hunk 1]
```diff
@@ -119,6 +120,9 @@
         this.courseAccessStorageService.onCourseAccessed(courseId);
     }
 
+    ngAfterViewInit() {
+        scrollToTopOfPage();
+    }
```
[the remark this answers is not in the document]
Comment (florian-glombik) [reply src/main/webapp/app/course/manage/course-management-tab-bar/course-management-tab-bar.component.ts:125]: I think this is a reasonable suggestion, could you please address it @edkaya ?

[the remark this answers is not in the document]
Comment (edkaya) [reply src/main/webapp/app/course/manage/course-management-tab-bar/course-management-tab-bar.component.ts:125]: Right, forgot to add it. Thanks 🙂

**Review by anian03 — APPROVED** [review]
Code looks good to me and works :)

**Review by sarpsahinalp — APPROVED** [review]
code lgtm

**Review by egekurt123 — APPROVED** [review]
Works as expected on ts3, and Code LGTM

**Review by rabeatwork — APPROVED** [review]
maintainer approve

**Review by krusche — APPROVED** [review]

**Pull request merged**